### PR TITLE
Remove LukasHirt from only office codeowners

### PR DESCRIPTION
**Pull request by LukasHirt**

### CODE REVIEW

**Review by kulmann — APPROVED** [review]
😞

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1 @@
-/onlyoffice/ @LukasHirt
 .drone.star @wkloucek @micbar @phil-davis @individual-it
```
Comment (pascalwengerter) [inline .github/CODEOWNERS:1]:
```suggestion
/accounts/ @kulmann @pascalwengerter 
/onlyoffice/ @kulmann @pascalwengerter 
/settings/ @kulmann @pascalwengerter 
/web/ @kulmann @pascalwengerter 
.drone.star @wkloucek @micbar @phil-davis @individual-it
```

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Since you're touching this already maybe bring it up to date some more? :)

**Review by pascalwengerter — APPROVED** [review]
💟

**Pull request merged**